lib/pact_broker/api/decorators/label_decorator.rb

```diff
@@ -11,20 +11,25 @@ class LabelDecorator < BaseDecorator
 
         include Timestamps
 
-        link :self do | options |
-          {
-            title: "Label",
-            name: represented.name,
-            href: label_url(represented, options[:base_url])
-          }
-        end
+        def to_hash(options)
+          hash = super
```

Comment:
adding some notes on why we are overriding this method from base class will be helpful for future